src/CAREUI/icons/CustomIcons.tsx

```
@@ -515,3 +515,58 @@ export const BedUnavailableSelected: React.FC<IconProps> = ({
 );
+
+export const TestTubeIcon: React.FC<IconProps> = ({ className = "size-4" }) => (
+  <svg
+    width="48"
+    height="48"
+    viewBox="0 0 48 48"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    className={className}
+    aria-label="TestTube Icon"
+  >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)